snuba/cli/consumer.py

```diff
@@ -139,21 +140,33 @@ def consumer(
     setup_logging(log_level)
     setup_sentry()
     logger.info("Consumer Starting")
-    storage_key = StorageKey(storage_name)
+    storage_keys = [
+        getattr(StorageKey, storage_name.upper()) for storage_name in storage_names
+    ]
 
-    metrics = MetricsWrapper(
-        environment.metrics,
-        "consumer",
-        tags={"group": consumer_group, "storage": storage_key.value},
-    )
-    configure_metrics(StreamMetricsAdapter(metrics))
+    if len(storage_keys) == 1:
```

Comment:
I don't think you need this if/else statement. Why don't you join all the storages together?

Reply:
That could be possible. The reason why I didn't join them all was because of the code snippet from multistorage_consumer.py (https://github.com/getsentry/snuba/blob/master/snuba/cli/multistorage_consumer.py#L235-L242) that shows how the multistorage case would create a MetricsWrapper